square/builder.go

```diff
@@ -129,7 +128,7 @@ func (b *Builder) Export() (Square, error) {
 	// interblob padding used when the blobs are correctly ordered instead of using worst case padding.
 	ss := inclusion.BlobMinSquareSize(b.currentSize)
 
-	// Sort the blobs by namespace. This uses SliceStable to preserve the order
+	// Sort the blobs by shares. This uses SliceStable to preserve the order
```

Comment:
[nit] revert because this sorts by namespace. 

```suggestion
	// Sort the blobs by namespace. This uses SliceStable to preserve the order
```